### DISCUSSION TITLE
2023.5.21.0

### DISCUSSION THREAD

**Discussion by BornToBeRoot** [BornToBeRoot/NETworkManager#2283] [Releases]
[:page_facing_up: Changelog](https://github.com/BornToBeRoot/NETworkManager/blob/7771ede38c6e7356e06e19db37bfd8b0ff2a8b8d/docs/Changelog/next-release.md)
[:book: Documentation](https://borntoberoot.net/NETworkManager/Documentation/Application)

This version mainly contains bug fixes 🐛  and documentation 📖 

---

| File | Checksum [SHA256]
|---|---|
|[Setup](https://github.com/BornToBeRoot/NETworkManager/releases/download/2023.5.21.0/NETworkManager_2023.5.21.0_Setup.exe)| `9444F9E0E5FE4B71BFF7C2A0C82DE1389B0BCBC32435CCA75510606A3B8A9D6C` |
|[Portable](https://github.com/BornToBeRoot/NETworkManager/releases/download/2023.5.21.0/NETworkManager_2023.5.21.0_Portable.zip)| `BE8C476386862AFED453C1F8F967765C5B90D9CDC61381A47B1B3C66C0B3E740` |
|[Archive](https://github.com/BornToBeRoot/NETworkManager/releases/download/2023.5.21.0/NETworkManager_2023.5.21.0_Archive.zip)| `0E838E4D6EBD1AE765E72997A96490BB330C346C73A87BC05FD94CE2EB7EE9B8` |

**System requirements**
- Windows 10 / Server x64 (1809 or later)
- [.NET Desktop Runtime 6.x (LTS)](https://dotnet.microsoft.com/download/dotnet/6.0)
- [Microsoft Edge - WebView2 Runtime](https://developer.microsoft.com/en-us/microsoft-edge/webview2/)

<hr /><em>This discussion was created from the release <a href='https://github.com/BornToBeRoot/NETworkManager/releases/tag/2023.5.21.0'>2023.5.21.0</a>.</em>

**Comment by xPretoo** [comment]
There is a way to save Remote Desktop and PuTTY credential? Is it possible to implement this function?
Will a function to monitor servers be implemented? Like the status of the disks, errors etc?

**Reply by BornToBeRoot** [reply]
Hi @xPretoo,

you can save them for Remote Desktop per Profile or Group, but not for PuTTY (this would be insecure and you should use something like KeePass). See: https://github.com/BornToBeRoot/NETworkManager/discussions/1671

I don't plan anything to monitor disks, errors, etc. You can use SNMP to get the values, but i would recommend a system like Nagios/Icinga2/Centreon/CheckMK...

**Reply by xPretoo** [reply]
Thanks. But I wrote this comment because under Remote Desktop it doesn't let me save my credentials anyway. It just asks me for the IP.

**Reply by BornToBeRoot** [reply]
Did you select the tab Remote Desktop?

![image](https://github.com/BornToBeRoot/NETworkManager/assets/16019165/3bcd953a-2171-4a6c-91b5-52e233e37086)

**Comment by xPretoo** [comment]
Yes, but when I try to connect to a server without a domain it tells me that the credentials are wrong.

**Reply by BornToBeRoot** [reply]
Strange. I cannot currently reproduce this.

Is your client & server member of the same domain? I'll try to reproduce and test it the days.

**Reply by xPretoo** [reply]
I figured out where it fits.
I set the user "administrator" with the password of the local administrator user and it fails me.
But if I put the password of the domain administrator user, even if I don't specify the domain, it enters me.
Is there any way to force the .\ or something like that?

**Reply by BornToBeRoot** [reply]
Maybe you can try to enter the IP/ComputerName (or a `. `) as domain.

The domain is only passed when it's set. It should behave the same like mstsc.exe.

**Reply by xPretoo** [reply]
With "." it was not taking me as a valid domain. With the IP it works fine. THANK YOU

**Reply by BornToBeRoot** [reply]
@xPretoo in the next release you can also use `.` in the domain field. #2305

**Reply by xPretoo** [reply]
Can you explain me why this doesn't work?
I can't save.
![Untitled](https://github.com/BornToBeRoot/NETworkManager/assets/13647658/69deaf69-72b9-46f6-ae6a-99a54166a6f3)

**Reply by BornToBeRoot** [reply]
Maybe you have an issue in the general tab.

**Reply by xPretoo** [reply]
Yes, excuse me. My fault.
Is there any way in WebConsole to save credentials to log in automatically?

**Reply by BornToBeRoot** [reply]
Thats currently not possible.

**Reply by BornToBeRoot** [reply]
> Can you explain me why this doesn't work? I can't save. ![Untitled](https://user-images.githubusercontent.com/13647658/241131080-69deaf69-72b9-46f6-ae6a-99a54166a6f3.png)

Tabs now have an indication if there is an error on the page:

![image](https://github.com/BornToBeRoot/NETworkManager/assets/16019165/26cab677-1d25-4b0d-9b51-34723ef22635)
